composer.json

```diff
@@ -33,7 +33,8 @@
         "ext-json": "*",
         "psr/http-message": "^1.0",
         "psr/http-client": "^1.0",
-        "guzzlehttp/guzzle": "^6.3|^7.0.1"
+        "guzzlehttp/guzzle": "^6.3|^7.0.1",
+        "aws/aws-sdk-php": "^3.185"
```

Comment:
I do not want aws-sdk-php to become part of the package dependencies, as it's not required for the functionality of the package, but is rather an extension. Maybe we should it to the `suggest` section instead.

Reply:
@mghoneimy Thank you for a feedback. I agree it's not a good idea to add this package to require section as it could be not needed. Then another question: I can't cover the the aws auth with test if I don't have the package installed. So, are we good to add it to require-dev?